AstronomyTests/AstronomyTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  AstronomyTests.swift
+//  AstronomyTests
+//
+//  Created by Lydia Zhang on 4/20/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import XCTest
+@testable import Astronomy
+
+struct MockDataLoader: NetworkDataLoader {
+    func loadData(from request: URLRequest, completion: @escaping (Data?, Error?) -> Void) {
+        DispatchQueue.global().asyncAfter(deadline: .now() + 2) {
```

Comment:
Cleaver use of a background queue here to simulate the network request's environment!